src/test/java/seedu/address/testutil/pet/TypicalPets.java

```diff
@@ -34,27 +33,27 @@
 
     public static final Pet AMY = new PetBuilder().withName("Amy")
             .withSpecies("Dog").withDateOfBirth("1/6/2015")
-            .withGender(Gender.FEMALE).withFoodList("Brand A:10")
+            .withGender("female").withFoodList("Brand A:10")
             .withTags("darkFur").build();
     public static final Pet BOB = new PetBuilder().withName("Bob")
             .withSpecies("Cat").withFoodList("Brand B:10")
-            .withDateOfBirth("19/6/1978").withGender(Gender.MALE)
+            .withDateOfBirth("19/6/1978").withGender("MALE")
             .withTags("fat", "lazy").build();
-    public static final Pet CARL = new PetBuilder().withName("Carl").withGender(Gender.MALE)
+    public static final Pet CARL = new PetBuilder().withName("Carl").withGender("MALE")
             .withDateOfBirth("1/3/2015").withSpecies("Husky").withFoodList("Brand C:10").build();
-    public static final Pet DANIEL = new PetBuilder().withName("Daniel").withGender(Gender.MALE)
+    public static final Pet DANIEL = new PetBuilder().withName("Daniel").withGender("MALE")
             .withDateOfBirth("2/3/2015").withSpecies("Dolphin").withTags("pink").withFoodList("Brand D:10").build();
-    public static final Pet ELLE = new PetBuilder().withName("Elle").withGender(Gender.FEMALE)
+    public static final Pet ELLE = new PetBuilder().withName("Elle").withGender("FEMALE")
             .withDateOfBirth("3/3/2015").withFoodList("Brand E:10").withSpecies("Parrot").build();
-    public static final Pet FIONA = new PetBuilder().withName("Fiona").withGender(Gender.FEMALE)
+    public static final Pet FIONA = new PetBuilder().withName("Fiona").withGender("FEMALE")
             .withDateOfBirth("4/3/2015").withFoodList("Brand F:10").withSpecies("Goldfish").build();
-    public static final Pet GEORGE = new PetBuilder().withName("George").withGender(Gender.MALE)
+    public static final Pet GEORGE = new PetBuilder().withName("George").withGender("MALE")
             .withDateOfBirth("5/3/2015").withSpecies("Slug").withFoodList("Brand G:10").build();
 
     // Manually added
-    public static final Pet HOON = new PetBuilder().withName("Hoon").withGender(Gender.MALE)
+    public static final Pet HOON = new PetBuilder().withName("Hoon").withGender("male")
             .withDateOfBirth("6/3/2015").withSpecies("passerine").build();
-    public static final Pet IDA = new PetBuilder().withName("Ida").withGender(Gender.MALE)
+    public static final Pet IDA = new PetBuilder().withName("Ida").withGender("MALE")
```

Comment:
I'm a little confused. So why is gender defined as all Upper case and used as sometimes lower cases, sometimes upper cases in test case? What exactly would the behaviors be anyways.

Reply:
I modified it such that it can be used as any letter case combination.e.g. user input can be "male", "MALE", "Male", etc.  It will eventually be converted to all upper case at Gender class. The test is to ensure any combination works.

I made this change because I feel it is a bit unnatural to enter upper case only for g/ ("MALE"). in the command line.

Reply:
I see. Remember to update the user guide then. Since there has been quite some confusions on the gender implementation. let's finalize it as the way you describe.